### PR TITLE
March updates

### PR DESCRIPTION
**Pull request by ajander**
I made some changes to the web dev instructions based on feedback from last Saturday's session. Please review by March 21. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,59 +40,59 @@ Tip: As you make updates on your website, be sure to _save_ frequently in Codean
...
 
         ```
         <h2>Visit us at:</h2>
-        <p>Code Koalas</p>
-        <p>1712 Main Street, #100</p>
-        <p>Kansas City, MO 64108</p>
+        <p>Plexpod</p>
+        <p>300 E. 39th St.</p>
+        <p>Kansas City, MO 64111</p>
         ```
```
Comment (Patrishjro) [inline README.md]: Nice update! :)

File: README.md [hunk 2]
```diff
@@ -18,25 +18,20 @@ To create a web site, we use a couple basic things - HTML, CSS, and JavaScript.
...
-1.  Log into Github. 
-2.  Go to https://github.com/kansascitywomenintechnology/codingandcupcakes and hit the fork button in the top right.
-3.  Log into Codeanywhere. 
-4.  In the Codeanywhere editor, select **File** -> **New Connection** -> **GitHub**.
-5.  Click **Connect your GitHub account**, enter your GitHub account details, and click **Authorize application**. 
-    *  You may need to disable your browser's pop-up blocker. To do so, click on the pop-up blocker icon at the far right of your browser's address bar and select "Always allow pop-ups from codeanywhere.com"
-6.  Select your codingandcupcakes repository and click Next. 
-    *  The application may take up to 45 seconds to copy the repository. When it's finished, you should see a list of files in the left sidebar of the editor screen. 
+Log into Github.
+1.  Go to https://github.com/kansascitywomenintechnology/codingandcupcakes and hit the **fork** button in the top right. This copies the codingandcupcakes code into a repository in your GitHub account.
```
Comment (Patrishjro) [inline README.md]: I like the added explanation here.

File: README.md [hunk 3]
```diff
@@ -45,59 +40,59 @@ Tip: As you make updates on your website, be sure to _save_ frequently in Codean
...
     *  Use paragraph tags to insert your contact information, and anything else you want to say. Note: It’s okay to erase the content between those tags.
-    *  You can use the Code Koalas address:
+    *  You can use the Plexpod address:
 
         ```
         <h2>Visit us at:</h2>
-        <p>Code Koalas</p>
-        <p>1712 Main Street, #100</p>
-        <p>Kansas City, MO 64108</p>
+        <p>Plexpod</p>
```
Comment (Patrishjro) [inline README.md]: Nice address update! :)

**Comment by cschneide143** [comment]
Looks good; thanks Adrienne.

File: README.md [hunk 4]
```diff
@@ -45,59 +40,59 @@ Tip: As you make updates on your website, be sure to _save_ frequently in Codean
...
- 
+
 ## Site Basics
 1.  Change the **title** of your site to **{your name}'s cupcake shop**.
 2.  Change the **h1** tag of the website to **{your name}'s cupcake shop**.
-3.  Add your own text about your cupcake shop using **h2**, **h3**, and **p** tags. Get creative in telling your story! 
+3.  Add your own text about your cupcake shop using **h2**, **h3**, and **p** tags. Get creative in telling your story!
 4.  Put in an image of your cupcakes on the page. (Look in the **img/cupcakes** folder for some recommendations. Don't see any you like? Search cupcake images on Google or Flickr.)
-    *  Hint: Images are stored between lists tags (```<li>``` and ```</li>```) in the code. Find these tags and replace the content so that it points to your images.
+    *  Hint: Images are stored between list tags (```<li>``` and ```</li>```) in the code. Find these tags and replace the content so that it points to your images.
```
Comment (tehfedaykin) [inline README.md]: perfect!

File: README.md [hunk 5]
```diff
@@ -110,8 +105,6 @@ Next we are going to add a form onto the Contact page, allowing people to email
 Note: You only need to type the **git push origin master:gh-pages** command once when you initially publish your website. If you make updates to your website at home and want to publish the changes, you can stop after **git push origin master**.
 
 ## Homework
-Did you love working with code? If you want to learn more on your own, we have suggestions for you. 
+Did you love working with code? If you want to learn more on your own, we have suggestions for you.
 1.  Create an account on http://www.codecademy.com/
 2.  Complete this course on Codecademy: http://www.codecademy.com/courses/web-beginner-en-zmn0b/0/1?content_from=make-an-interactive-website%3Ayour-first-program
-
```
Comment (tehfedaykin) [inline README.md]: What about adding instructions to fill out the survey here too?

Comment (ajander) [reply README.md]: I think that's a good idea. Right now, though, our survey has language specific to Jewelbots: https://docs.google.com/forms/d/e/1FAIpQLSez_eKdTOfZsfv_IrxAFOaW20Hp5o7q05uosXrGMw_2OTepIQ/viewform 

I think we could make it generic so it works for both sessions, and after that I can add a link to it at the end of the Web Dev worksheet. Who does the editing of the survey? @cschneide143 ?

**Pull request merged**